blog/2025/01/conference.md

```diff
@@ -0,0 +1,82 @@
+Later in the afternoon, Terence Eden captivated and inspired the audience with lessons from open-sourcing the UK's COVID-19 tracing app. His story of the time showed all sides to open collaboration in government projects, emphasising how these principles (embraced in the service standard) can lead to better public trust and more robust software solutions and better security.
+<p style="clear:both;"></p>
+<img src="/assets/images/conference-2025/teden.jpg"
+     alt="Terence Eden standing near banner at the Cross Government Software Engineering Conference"
+     style="float:left; width:42%; height:auto; border:12px solid #ccc; margin:1em;" />
+<img src="/assets/images/conference-2025/asim.jpg"
+     alt="Asim Hussain standing near banner at the Cross Government Software Engineering Conference"
+     style="float:left; width:42%; height:auto;  border:12px solid #ccc; margin:1em;" />
+<p style="clear:both;"></p>
+
+### Engaging Lightning Talks
+
+One of the standout features of the conference came from the community members themselves , this was the series of lightning talks, where speakers from various government departments delivered the following topics :
+
+- Transforming API Management: From Vendor Lock-in to Open Source by Abhi Markan
+
+- So, You're Interested in Green Software - Now What? by Rachael Robinson
+
+- Creative Technology in Government by Brendan Arnold
+
+- Data Engineering is a Software Discipline by Michal Poreba
+
+- We Ran a Hackathon: Here's How to Make It Work by Hettie Street and Stephanie De Jong
+
+
+### **Great Sessions**
+
+Beyond the lightning talks, the conference provided in-depth sessions that explored critical government software initiatives. Topics included:
+
+- Transforming How We Implement the Rules of Legal Aid by Bex Appleyard
+
+- GOV.UK One Login: What, Why and How by Phil Fordham & Paul Dougan
+
+- Applying AI to the Legislative Process by Liam Wilkinson
+
+- Evolving GOV.UK Chat from Prototype to Production AI System by Alessia Tosi, Kevin Dew, and Rich Eveson
+
+These sessions provided a service view, from the teams themselves a real deep look at the innovative work happening across government departments, offering attendees practical takeaways to improve their own projects.
+
+### Building Connections and Community
+
+As is always a valuable aspect of the conference being the opportunity to network. With attendees from 36 different departments and agencies, the event hopefully will create new connections between people who might otherwise not meet. There seemed to be people lost in discussions, sharing challenges, and exchanging best practices, the objectives of a cross-government software engineering community were surely met.
```

Comment:
```suggestion
The conference also provided valuable time for attendees to network and chat. With attendees from 36 different departments and agencies, the event hopefully created new connections between people who might otherwise not meet. At lunch there seemed to be people lost in discussions, sharing challenges, and exchanging best practices, the objectives of a cross-government software engineering community were surely met.
```